lib/api/src/grpc/google.protobuf.rs

```diff
@@ -16,8 +16,8 @@ pub struct Struct {
 }
 /// `Value` represents a dynamically typed value which can be either
 /// null, a number, a string, a boolean, a recursive struct value, or a
-/// list of values. A producer of value is expected to set one of these
-/// variants. Absence of any variant indicates an error.
+/// list of values. A producer of value is expected to set one of that
+/// variants, absence of any variant indicates an error.
```

Comment:
`these` was correct :smiley: 

```suggestion
/// list of values. A producer of value is expected to set one of these
/// variants, absence of any variant indicates an error.
```